src/items/generics.md

```diff
@@ -40,21 +40,21 @@ referred to with path syntax.
 
 ## Where clauses
 
-> **<sup>Syntax</sup>**  
-> _WhereClause_ :  
+> **<sup>Syntax</sup>**
+> _WhereClause_ :
```

Comment:
These spaces are needed for line breaks. They could be replaced with `<br>`s if you prefer.